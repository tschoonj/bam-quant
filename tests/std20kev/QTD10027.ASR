File: C:\xrfdaten\2014KW30\chris\std20kev\std10027.spe
Start: 21.07.2014 18:09:20
Stop: 21.07.2014 18:11:00
BESSY: 298.6760689 298.894711
IONI: 3.611937E-11 3.500453E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 81235
Run: 100000
Dead: 19
ICR: 59250
OCR: 0
Accumulated BESSY: 59657.4771012
Accumulated Ioni: 7.10377032E-09
N Accumulation: 200
Normfactor:  3.4657553e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Co
exp_x 71.2603988647461 71.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  27   1      6.925   4158931.      2039.     154.67
