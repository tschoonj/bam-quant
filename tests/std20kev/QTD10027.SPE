$DATA:
0 2000
19232
12299
5195
1680
363
50
17
6
2
4
1
0
0
0
1
3
124
1034
1345
1225
1210
1186
1076
1094
1023
1005
1019
878
913
942
843
824
842
818
808
796
760
797
717
715
726
728
679
681
698
654
668
644
640
611
645
623
665
622
671
555
605
601
565
578
598
596
564
596
575
568
576
539
556
542
548
541
561
532
552
498
515
500
533
474
503
534
533
459
503
489
502
523
483
506
502
511
502
508
580
486
509
506
496
554
544
594
590
559
516
516
509
506
452
530
488
479
493
492
437
468
472
436
475
454
471
456
423
460
465
499
451
481
482
516
484
477
456
507
476
452
489
445
485
452
442
460
474
560
577
636
651
626
611
580
498
478
444
437
457
457
498
454
450
441
454
511
445
445
445
433
480
471
447
499
405
432
442
433
449
430
416
456
433
467
488
440
517
472
454
466
450
439
443
424
484
431
446
444
440
450
444
453
455
486
526
475
456
470
456
452
439
492
475
461
480
481
487
518
457
448
455
483
489
453
487
508
459
457
485
451
480
522
540
511
518
461
566
563
527
523
553
527
564
530
571
516
570
553
588
559
571
587
612
720
793
924
1215
1445
1798
2079
2115
2107
1955
1585
1260
1046
868
712
710
633
706
676
738
718
645
699
701
696
685
722
714
696
659
709
716
784
788
809
780
789
919
872
903
980
1063
1064
1044
1090
1098
1089
1028
1028
950
971
943
940
986
915
975
977
988
990
1044
1044
1062
1126
1127
1188
1170
1286
1288
1353
1386
1492
1636
1552
1691
1796
1953
2058
2272
2451
2656
3062
3458
4275
5614
8831
15649
29589
55608
99776
165726
253088
351032
440890
502778
520009
484460
409471
312819
216843
136176
77734
40686
19952
9136
4499
2631
1695
1476
1433
1368
1344
1282
1410
1450
1501
1661
1780
2146
2482
3047
4120
5883
9098
14084
22467
33577
46769
61166
72343
78799
78654
72053
59384
45174
31649
20040
11799
6215
3013
1468
631
278
145
88
54
55
49
50
45
51
41
39
29
40
30
28
22
32
26
28
31
28
27
26
21
21
21
25
22
16
21
17
12
30
21
18
15
19
13
19
18
21
22
14
19
14
14
14
15
6
11
12
16
12
16
19
6
13
15
17
13
6
14
14
9
14
19
8
16
13
12
12
10
13
10
15
18
10
12
13
13
8
13
11
10
14
15
16
11
16
12
6
10
14
5
6
11
17
11
8
11
7
12
17
10
8
8
12
11
7
9
12
13
8
9
10
8
11
7
3
14
7
5
5
16
16
10
14
11
13
6
8
4
13
6
15
5
10
4
3
12
10
11
10
7
14
5
9
12
10
8
9
7
9
11
8
9
9
12
5
6
8
9
10
10
9
8
9
7
8
7
9
7
5
5
6
8
10
6
8
9
7
6
7
6
6
10
11
13
9
10
3
7
7
8
7
10
11
9
10
6
15
11
13
14
16
14
14
29
20
16
27
11
13
14
15
9
13
10
7
5
9
5
7
8
8
4
14
7
12
6
10
9
13
15
9
13
12
7
12
11
6
16
13
9
11
14
9
10
14
10
7
13
10
14
9
13
14
8
14
12
17
9
13
13
10
13
18
16
15
17
17
14
22
16
26
20
25
26
16
28
33
36
38
88
92
155
234
351
488
702
967
1209
1361
1546
1818
1801
1835
1582
1426
1222
936
700
505
331
244
128
86
65
35
26
21
19
11
19
15
24
27
25
37
43
75
86
118
144
201
272
334
410
478
528
571
558
541
504
459
354
299
215
142
110
60
44
34
19
7
10
2
7
3
4
1
4
4
6
7
9
8
10
12
17
16
28
24
44
29
38
46
56
45
34
44
38
20
19
9
7
7
3
8
3
7
11
12
9
13
19
15
29
32
28
25
32
23
17
16
17
15
9
10
4
2
3
5
0
1
0
4
3
2
3
1
2
0
0
1
0
3
1
2
3
0
1
2
1
1
2
2
1
4
3
5
2
3
3
1
0
1
2
1
3
1
1
2
2
4
1
3
1
1
0
1
0
0
4
2
2
2
0
2
1
1
3
3
1
4
2
5
2
4
5
1
4
1
5
1
2
4
1
3
6
10
7
5
9
6
11
10
4
7
1
6
1
5
4
3
2
5
4
8
4
3
3
6
5
5
5
8
4
6
6
6
5
4
8
5
8
5
7
12
4
4
8
6
9
11
7
10
10
8
11
7
8
11
16
8
16
12
13
14
12
19
10
18
12
17
16
22
15
25
23
24
24
19
24
22
24
29
25
28
27
29
33
16
45
29
40
38
40
31
34
50
38
35
41
46
37
39
39
46
51
64
62
49
57
65
59
57
54
60
87
77
89
122
112
122
110
113
131
127
123
133
141
150
131
122
120
109
98
90
64
75
69
54
45
53
34
26
20
9
19
4
7
2
0
5
2
4
1
0
3
2
5
5
0
2
5
11
6
4
3
1
3
7
4
1
0
1
0
1
0
0
0
0
0
0
0
0
0
0
0
1
1
0
0
0
2
0
3
1
3
3
3
3
3
0
1
2
2
0
2
0
1
1
0
1
0
0
0
0
0
0
0
0
2
1
0
0
0
0
0
0
0
0
0
2
0
0
2
0
0
0
1
0
0
0
0
2
0
1
0
0
0
0
0
0
1
0
0
0
1
0
0
0
1
0
0
0
0
0
0
0
0
2
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
1
0
0
0
0
1
0
1
0
0
0
0
0
0
1
1
0
0
0
0
0
0
0
0
0
2
1
0
0
1
0
1
0
0
0
1
0
0
0
0
0
1
2
2
0
0
1
1
0
0
0
1
0
0
0
0
0
0
0
0
1
2
1
1
0
2
2
0
2
2
0
0
3
0
1
4
0
0
0
1
1
1
0
1
0
1
1
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
1
0
0
0
0
2
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
1
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
0
File: C:\xrfdaten\2014KW30\chris\std20kev\std10027.spe
Start: 21.07.2014 18:09:20
Stop: 21.07.2014 18:11:00
BESSY: 298.6760689 298.894711
IONI: 3.611937E-11 3.500453E-11
ScanMotor1 exp_x: 1 1
ScanMotor2 exp_y: 1 1
Live: 81235
Run: 100000
Dead: 19
ICR: 59250
OCR: 0
Accumulated BESSY: 59657.4771012
Accumulated Ioni: 7.10377032E-09
N Accumulation: 200
Normfactor:  3.4657553e+008
E_cal:      0.020000000
E_cal_offset:      0.000000
Comments: Co
 
exp_x 71.2603988647461 71.2604
exp_y 122.323806762695 122.3238
exp_z 53.8899993896484 53.89
exp_w 135 135
exp_Tisch 870.6 870.600001928
dcm_theta 0 0
det_y 52.85 52.85
dmm_energy 20 20
dcm_energy 23 -184976.74536672
S3_horSize 0.03 3.00000000000001E-02
S3_verSize 0.03 3.00000000000001E-02
$PEAKS:   
      1
  27   1      6.925   4158931.      2039.     154.67
