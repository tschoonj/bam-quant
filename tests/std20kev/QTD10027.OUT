     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  AXIL IBM-PC V3.00                       10-08-2014   14:23:41  บ   
     บ  Spectrum: QTD10027.SPE                                     0.s บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ  Fitting Region: channels 100 -1980        4 iterations done    บ   
     บ  ChiSquare =   107.2   last change =   .04%    lambda= 1.E-06   บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                        CALIBRATION  DATA                        บ   
     ฬออออออออออออออออหออออออออออออออออออออออออหอออออออออออออออออออออออน   
     บ                บ   Initial estimate     บ    Final estimate     บ   
     ฬออออออออออออออออฮออออออออออออออออออออออออฮอออออออออออออออออออออออน   
     บ  ZERO (eV)     บ      -46.4 ๑   100.0   บ     -46.4 ๑      .5   บ   
     บ  GAIN (eV/ch)  บ     20.280 ๑   2.000   บ    20.280 ๑    .001   บ   
     บ det NOISE (eV) บ      -61.9 ๑    40.0   บ     -61.9 ๑    -4.1   บ   
     บ  FANO factor   บ       .188 ๑    .050   บ      .188 ๑    .002   บ   
     ศออออออออออออออออสออออออออออออออออออออออออสอออออออออออออออออออออออผ   
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                           PEAK  DATA                            บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ  # Line       E(KeV)  rel. int.   peak area   st. dev    chi-sq บ   
     บ               chan#   fwhm (eV)      backgr            tot. abs บ   
     ฬอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออน   
     บ                                                                 บ   
     บ  1 Co-Ka                           4158931. ๑   2039.           บ   
     บ    KA1         6.925   1.00000     4158931. ๑   2039.    154.7  บ   
     บ              343.763    155.01         204.           9.34E-01  บ   
     บ                                                                 บ   
     บ  2 Co-Kb                            677805. ๑    824.           บ   
     บ    KB1         7.649   1.00000      677805. ๑    824.     47.4  บ   
     บ              379.464    164.14         185.           9.50E-01  บ   
     บ                                                                 บ   
     บ  3 Pile_up                           26088. ๑    164.           บ   
     บ    L1N3       13.850    .73910       19282. ๑    122.     39.1  บ   
     บ              685.239    227.78          95.           9.86E-01  บ   
     บ    L1N2       14.574    .24122        6293. ๑     41.      6.7  บ   
     บ              720.940    234.09          82.           9.87E-01  บ   
     บ    L1O3       15.298    .01968         513. ๑      9.      1.9  บ   
     บ              756.641    240.23          77.           9.87E-01  บ   
     บ                                                                 บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
     ษอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออป   
     บ                         FILTER BACKGROUND                       บ   
     ศอออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออออผ   
                                                                           
